src/chart/rendering/drawlegend.h

```diff
@@ -1,36 +1,27 @@
 #ifndef CHART_RENDERING_DRAWLEGEND_H
 #define CHART_RENDERING_DRAWLEGEND_H
 
-#include "base/geom/rect.h"
-#include "base/gfx/canvas.h"
-#include "chart/generator/plot.h"
-#include "chart/main/events.h"
-#include "chart/main/style.h"
+#include "chart/rendering/drawingcontext.h"
 
 namespace Vizzu
 {
 namespace Draw
 {
 
-class drawLegend
+class drawLegend : public DrawingContext 
 {
 public:
-	drawLegend(const Geom::Rect &rect,
-	    const Gen::Plot &plot,
-	    const Events::Draw::Legend &events,
-	    Gfx::ICanvas &canvas,
+	drawLegend(const DrawingContext &context,
 	    Gen::ChannelId channelType,
 	    double weight);
 
 private:
 	Geom::Rect contentRect;
-	const Gen::Plot &plot;
 	const Events::Draw::Legend &events;
```

Comment:
this member hides the `DrawingContext::events` member.

Reply:
fixed.